backend/queue/rmq.go

```diff
@@ -0,0 +1,64 @@
+package queue
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/adjust/rmq/v2"
+	"github.com/pace/bricks/backend/redis"
+	"github.com/pace/bricks/maintenance/health/servicehealthcheck"
+)
+
+var (
+	rmqConnection     rmq.Connection
+	queueHealthLimits map[string]int
```

Comment:
Since we have two different go routines that can read/write we should use `sync.Map`

Reply:
Done